packages/slice-machine/src/modules/availableCustomTypes/index.ts

```diff
@@ -226,6 +256,49 @@ export function* renameCustomTypeSaga({
   }
 }
 
+export function* deleteCustomTypeSaga({
+  payload,
+}: ReturnType<typeof deleteCustomTypeCreator.request>) {
+  try {
+    yield call(deleteCustomType, payload.customTypeId);
+    yield put(deleteCustomTypeCreator.success(payload));
+    yield put(
+      modalCloseCreator({ modalKey: ModalKeysEnum.DELETE_CUSTOM_TYPE })
```

Comment:
This happens in both cases (try and except), you could put it just once at the end